src/pipelines.py

```diff
@@ -4,8 +4,8 @@
 from . import loaders
 from .steps.base import Step, Dummy
 from .steps.preprocessing.misc import XYSplit
-from .utils import squeeze_inputs, make_apply_transformer
-from .models import PyTorchUNet, PyTorchUNetWeighted
+from .utils import squeeze_inputs, make_apply_transformer_stream
```

Comment:
@taraspiotr it seems that the returning stream mode took reins over the old `make_apply_transformer` completely. I think the re-return of the non stream mode `make_apply_transformer` could make things easier. Let's see if they can coexist. 

What I really mean is that I would rather have `make_apply_transformer` and `make_apply_transformer_stream` as 2 functions to make the distinction more vivid. I must have not communicated that before, sorry.

Reply:
@jakubczakon you have these two functions in utils.py, `make_apply_transformer_stream` wraps `make_apply_transformer` adding stream mode option. I think it's cleaner, because you don't have to pass twice the same arguments in pipelines.py for each step, for example this would have to be typed twice:
```
make_apply_transformer_stream(post.resize_image,
output_name='resized_images',
apply_on=['images', 'target_sizes'],
```

Reply:
@taraspiotr oh ok, so the only problem remaining is that we are using only make_apply_transformer_stream in the pipelines.py wheras most of the time we could use the simple make_apply_transformer . Do you strongly thing that using a flag is better?

Reply:
@taraspiotr ok I see now. let's stay with the flag

Reply:
@taraspiotr you conviced me with doulbe arguments. Let's stay with the flag